app/src/main/java/com/babylon/wallet/android/presentation/transaction/composables/RawManifestView.kt

```diff
@@ -1,22 +1,71 @@
 package com.babylon.wallet.android.presentation.transaction.composables
 
-import androidx.compose.foundation.text.selection.SelectionContainer
+import android.content.ClipData
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
 import androidx.compose.material.Text
+import androidx.compose.material3.Icon
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.text.font.Typeface
 import androidx.compose.ui.unit.sp
+import androidx.core.content.getSystemService
+import com.babylon.wallet.android.designsystem.R
 import com.babylon.wallet.android.designsystem.theme.RadixTheme
+import com.babylon.wallet.android.presentation.ui.modifier.throttleClickable
 
 @Composable
 fun RawManifestView(
     modifier: Modifier = Modifier,
     manifest: String
 ) {
-    SelectionContainer {
+    val context = LocalContext.current
+    Column(
+        modifier = modifier.fillMaxWidth(),
+        horizontalAlignment = Alignment.End
+    ) {
+        Row(
+            modifier = Modifier
+                .padding(RadixTheme.dimensions.paddingSmall)
+                .background(
+                    color = RadixTheme.colors.gray4,
+                    shape = RadixTheme.shapes.roundedRectSmall
+                )
+                .throttleClickable {
+                    context
+                        .getSystemService<android.content.ClipboardManager>()
```

Comment:
is `LocalClipboardManager.current.setText()` not compose equivalent to obtaining ClipboardManager?

Reply:
> LocalClipboardManager.current.setText()

Yes you absolutely right 👍